app/models/item.rb

```diff
@@ -103,6 +104,17 @@ class Item < ApplicationRecord
   }
 
   before_destroy :validate_destroy, prepend: true
+  before_create :set_reporting_category
+
+  enum :reporting_category, [
```

Comment:
Do we want to store the full string with spaces and capitals in the database? I feel like it'd be easier to work with if the DB value was a more "computerized" string. E.g. it makes it easier to understand the built-in methods like `item.adult_incontinence?`.

Reply:
Yea I was trying to make the string value stored human readable, which probably isn't the best approach. I refactored it to us an enum with integer values. This means that we will have to transform the reporting categories to be human readable (eg. use `titleize` to change "adult_incontinence" to "Adult Incontinence"). But this seems like a better separation of concerns. And we can easily add Spanish translations if needed.

Reply:
Ok I had to name the new enums a little odd. Because otherwise the old `adult_incontinence` scope would clash with the `adult_incontinence` class method created from the enum.